etstool.py

```diff
@@ -111,8 +111,8 @@
 
 # Required runtime dependencies. Should match install_requires in setup.py
 dependencies = {
+    "pyface",
     "traits" + TRAITS_VERSION_REQUIRES,
-    # pyface is installed from source via ci-src-requirements.txt
 }
 
 # NOTE : pyface is always installed from source
```

Comment:
The source_dependencies still needs to be updated so that cron job is run against pyface master?

Reply:
yep. nice catch. corrected.